lib/Secrets/doc/sailfish-secrets.cpp

```diff
@@ -0,0 +1,17 @@
+namespace Sailfish {
+namespace Secrets {}
+}
+
+/*!
+  \namespace Sailfish
+  \brief This namespace provides various APIs to interact with the Sailfish OS system.
+ */
```

Comment:
Is there any way to avoid duplicating this code and comments between secrets and crypto?

Reply:
Possibly, I could split out "sailfish.cpp" and move it to the top level doc/ directory, which contains just the namespace definition, and then include that one from "sailfish-secrets.cpp" and "sailfish-crypto.cpp" in the per-module doc/ directories.

Reply:
done